lib/tapioca/helpers/test/isolation.rb

```diff
@@ -0,0 +1,37 @@
+# typed: true
+# frozen_string_literal: true
+
+require "tapioca/ruby_ext/forking_patch"
+require "tapioca/ruby_ext/subprocess_patch"
+
+module Tapioca
+  module Helpers
+    module Test
+      # Copied from ActiveSupport::Testing::Isolation since we cannot require
+      # constants from ActiveSupport without polluting the global namespace.
+      module Isolation
+        extend T::Sig
+        require "thread"
+
+        sig { returns(T::Boolean) }
+        def self.forking_env?
+          !ENV["NO_FORK"] && Process.respond_to?(:fork)
+        end
+
+        def run
```

Comment:
This was the root cause of `Minitest` failing to run locally and on CI (Thanks to @RyanBrushett for figuring it out). This method should technically return `Mintest::Result`, however, doing that raised some type errors since it doesn't return a `Minitest::Result` and parsing into a `Minitest::Result` caused _every_ test to run in isolation causing some messy output. 

Took @paracycle's suggestion from Slack and just removed the signature from this specific method. It did require lowering the strictness on this file from `strict` to `true`. But the test suite is working as expected again.